app/models/ability.rb

```diff
@@ -7,7 +7,7 @@ class Ability
   attr_accessor :user, :user_id, :admin, :supervisor, :reviewer, :user_offer_ids, :user_permissions
 
   PERMISSION_NAMES = ['can_manage_items', 'can_manage_goodcity_requests', 'can_manage_packages',
-    'can_manage_offers', 'can_manage_organisations_users',
+    'can_manage_offers', 'can_manage_organisations_users', 'can_search_packages',
```

Comment:
I'm not sure we need a permission for being able to search as this is something an anonymous user can do. However, we do need to restrict what packages anonymous users can see (only `allow_web_publish == true`) which is something we do already.